### PR TITLE
Try/type rebased

### PR DESCRIPTION
**Pull request by jasmussen**
**Description**

A rebased version of #422. Pushed as a separate branch in case I made an error in the rebase.

### CODE REVIEW

File: patterns/cta-newsletter.php [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?php
 /**
  * Title: Newsletter sign-up
- * Slug: twentytwentyfive/cta-newsletter
+ * Slug: twentytwentyfive/newsletter-sign-up
```
Comment (juanfra) [inline patterns/cta-newsletter.php]:
```suggestion
 * Slug: twentytwentyfive/cta-newsletter
```

File: patterns/contact-centered-social-link.php [hunk 2]
```diff
@@ -12,15 +12,20 @@
  */
 
 ?>
-<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"var:preset|spacing|80","bottom":"var:preset|spacing|80"},"blockGap":"var:preset|spacing|50"}},"layout":{"type":"constrained"}} -->
+
+<!-- wp:group {"metadata":{"categories":["contact"],"patternName":"twentytwentyfive/contact-centered-social-link","name":"Centered link and social links"},"align":"full","style":{"spacing":{"padding":{"top":"var:preset|spacing|80","bottom":"var:preset|spacing|80"},"blockGap":"var:preset|spacing|50"}},"layout":{"type":"constrained"}} -->
```
Comment (carolinan) [inline patterns/contact-centered-social-link.php]:
```suggestion
<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"var:preset|spacing|80","bottom":"var:preset|spacing|80"},"blockGap":"var:preset|spacing|50"}},"layout":{"type":"constrained"}} -->
```

File: theme.json [hunk 3]
```diff
@@ -1413,43 +1401,45 @@
...
-				}
-			},
-			"h6": {
-				"typography": {
-					"fontSize": "var:custom|font-size|heading-6",
+					"fontSize": "var:preset|font-size|small",
 					"fontStyle": "normal",
 					"fontWeight": "700",
 					"letterSpacing": "1.4px",
 					"textTransform": "uppercase"
```
Comment (carolinan) [inline theme.json]: The uppercase and 700 font weight is meant to be on the H6, not H5.

Comment (carolinan) [reply theme.json]: This is the current result:

```
			"h5": {
				"typography": {
					"fontSize": "var:preset|font-size|small",
					"fontStyle": "normal",
					"fontWeight": "700",
					"letterSpacing": "1.4px",
					"textTransform": "uppercase"
				}
			},
			"h6": {
				"typography": {
					"fontSize": "var:preset|font-size|small",
					"letterSpacing": "0.5px"
				}
			},
```

File: patterns/contact-centered-social-link.php [hunk 4]
```diff
@@ -12,15 +12,20 @@
...
+	<!-- wp:group {"align":"wide","layout":{"type":"default"}} -->
 	<div class="wp-block-group alignwide">
-		<!-- wp:paragraph {"align":"center","style":{"typography":{"fontSize":"var(--wp--custom--font-size--huge)"}}} -->
-		<p class="has-text-align-center" style="font-size:var(--wp--custom--font-size--huge);"><?php echo wp_kses_post( _x( 'Got questions? <br><a href="#" rel="nofollow">Feel free to reach out.</a>', 'Heading of the Contact social link pattern', 'twentytwentyfive' ) ); ?></p>
+		<!-- wp:paragraph {"align":"center","className":"is-style-text-display","style":{"typography":{"fontStyle":"normal","fontWeight":"400"}}} -->
+		<p class="has-text-align-center is-style-text-display" style="font-style:normal;font-weight:400"><?php echo wp_kses_post( _x( 'Got questions? <br><a href="#" rel="nofollow">Feel free to reach out.</a>', 'Heading of the Contact social link pattern', 'twentytwentyfive' ) ); ?></p>
 		<!-- /wp:paragraph -->
 
-		<!-- wp:social-links {"iconColor":"contrast","className":"is-style-logos-only","layout":{"type":"flex","justifyContent":"center"}} -->
+		<!-- wp:spacer {"height":"var:preset|spacing|40","style":{"layout":{}}} -->
```
Comment (carolinan) [inline patterns/contact-centered-social-link.php]: Nit:
I believe the Layout is only needed on the spacer when display:flex is used.
The wrapping group was changed from a stack to a normal group.

**Comment by carolinan** [comment]
@richtabor @jasmussen In the twentytwentyfive/hero-podcast pattern, a heading was changed from an H1 to H2.
The reason it was set to an H1 was because it is the first heading on one of the landing pages, twentytwentyfive/page-landing-podcast.
This landing page must have an H1, be it this one or another, because it is intended to be used with "page not title" template.

**Comment by richtabor** [comment]
> @richtabor @jasmussen In the twentytwentyfive/hero-podcast pattern, a heading was changed from an H1 to H2.
The reason it was set to an H1 was because it is the first heading on one of the landing pages, twentytwentyfive/page-landing-podcast.
This landing page must have an H1, be it this one or another, because it is intended to be used with "page not title" template.

It's far more likely this will be used on existing pages, thus adding an H1 in the middle of the page. 

We cannot accurately assume the template that will be employed with this particular pattern. If anything, we assume the default page template is utilized and this pattern is leveraged individually.

**Pull request merged**